data-prepper-plugins/service-map-stateful/src/test/java/com/amazon/dataprepper/plugins/prepper/ServiceMapStatefulPrepperTest.java

```diff
@@ -172,21 +186,33 @@ public void testTraceGroups() throws Exception {
 
         Assert.assertTrue(evaluateEdges(relationshipsFound).containsAll(expectedSourceDests));
 
+        // Verify gauges
+        final List<Measurement> spansDbSizeMeasurement = MetricsTestUtil.getMeasurementList(
+                new StringJoiner(MetricNames.DELIMITER).add("testPipelineName").add("testServiceMapPrepper")
+                        .add(ServiceMapStatefulPrepper.SPANS_DB_SIZE).toString());
+        Assert.assertEquals(1, spansDbSizeMeasurement.size());
```

Comment:
I think you're not verifying the value of the metrics here right?

Reply:
Oh right, need to add that.